invoices_service.go

```diff
@@ -160,3 +160,23 @@ func (s *invoicesImpl) MarkFailed(invoiceNumber int) (*Response, *Invoice, error
 
 	return resp, &dst, err
 }
+
+// Allows custom invoice amounts to be refunded and generates a refund invoice.
```

Comment:
Updated in [e9b88a](https://github.com/blacklightcms/recurly/pull/11/commits/1b52bfba87d06be765b03b13c108236146e9b88a).